app/forms/form_navigation.rb

```diff
@@ -5,8 +5,10 @@ class FormNavigation
     CountyLocationController,
     WhereDoYouLiveController,
     SupportedCountyController,
-    NotYetSupportedCountyController,
+    NotYetSupportedController,
     ChangeTypeController,
+    SelfEmployedController,
+    NotYetSupportedController,
     TellUsAboutYourselfController,
```

Comment:
This is the first time we've had one controller multiple times in this list which might be a little weird in one or two places. We use this list of controller to build all our 'edit' routes and also to generate the 'screens' page where you can browse to any screen.

We could add a class method for 'unique_form_controllers' or something that only returns one of each controller and use that at least for building the routes.